src/components/TodoModal/TodoModal.tsx

```diff
@@ -5,32 +5,28 @@ import { getUser } from '../../api';
 import { User } from '../../types/User';
 
 type Props = {
-  selectTodo: () => Todo | undefined;
+  selectedTodo: Todo | undefined;
   viewModule: (id :number) => void
 };
 
-export const TodoModal: React.FC<Props> = ({ selectTodo, viewModule }) => {
+export const TodoModal: React.FC<Props> = ({ selectedTodo, viewModule }) => {
   const [selectedUser, setSelectedUser] = useState<User | null>(null);
 
-  const selectedTodo = selectTodo();
+  useEffect(() => {
+    getUser(selectedTodo?.userId)
```

Comment:
Here I  use  `selectedTodo?.userId` because  👇
![image](https://user-images.githubusercontent.com/108505176/194052927-8a64c962-3157-4214-9232-194004b57cbe.png)

Reply:
Sorry, but I don’t see any comments about selectedTodo?.userId